spec/fixtures/configs/eslintignorekey/.eslintignore

```diff
@@ -0,0 +1 @@
+**/filedoesntexist
```

Comment:
This file doesn't appear to be used?

Reply:
This file is required by spec `"will still lint the file if .eslintignore file"`.  This spec tries to lint `ignored.js` in this folder. When `.eslintignore` is found with a non-matching glob, `eslintIgnore` linting should occur, regardless of contents of `eslintIgnore` key.

Reply:
I'm not really sure why this should be the case. But I'm just following the docs, which state 
> **If an .eslintignore file is not found** and an alternate file is not specified, eslint will look in package.json for an eslintIgnore key to check for files to ignore.

And sure enough, that seems to be the behavior we're getting.

Reply:
Ah ha, so this is just checking that ESLint is behaving as it should.

Reply:
Yeah, I guess if you think about it it, this whole suite is redundant, because we're basically just saying `eslint` is doing for us what it says in the docs. Yet if you were to run this suite against the version of `linter-eslint` published to `apm`, it  would fail. So it seems like we should have it.